README.md

```diff
@@ -1 +1,106 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-microverse] <a name="about-project"></a>
+
+it is simple html and css project which display hello microverse in siliver text color
+
+**[your_project__name]** is a...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+ i use linter to check my code and html and css
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[it text color is silver]**
+- **[it use h1 tag]**
+- **[and use standard html and css]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+you  can fork and use it as you want 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As stated before kindly 😃add appropriate information in this section of getting started. for example, in the setup, you can give the link of your repo to clone.💯